pennylane/ops/qubit.py

```diff
@@ -2157,6 +2157,43 @@ def _matrix(cls, *params):
 
         return U
 
+    @staticmethod
+    def decomposition(U, wires):
+        # Decompose arbitrary single-qubit unitaries as the form RZ RY RZ
+        if U.shape[0] == 2:
```

Comment:
```suggestion
        if qml.math.shape(U)[0] == 2:
```

Since not all frameworks *cough* tensorflow *cough* return integers from the `shape` property